ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -5,13 +5,14 @@
     git_credentials_id: ANTREA_GIT_CREDENTIAL
     org_repo: antrea-io/antrea
     jobs:
-      - '{name}-{test_name}-for-period':
+      - '{name}-{test_name}-for-period-{jenkins_location}':
```

Comment:
I am not sure I got this, why job rename can enable the job update? 
I believe cloud and lab jobs are running in different Jenkins, so why is necessary to rename it?
@XinShuYang any idea?

Reply:
I agree there is no need to differentiate the updater job by name, as the Jenkins location has already been defined in the YAML.

Reply:
sure